statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -646,20 +828,35 @@ def bic(self):
     @cache_readonly
     def cov_params_default(self):
```

Comment:
It seems like it might be better if I kept this more closely in line with other models, and made this an attribute and somehow make this class work together with `get_robustcov_results`. What do you think?